src/main/flight/pid_init.c

```diff
@@ -316,8 +316,8 @@ void pidInitConfig(const pidProfile_t *pidProfile)
     pidRuntime.crashTimeDelayUs = pidProfile->crash_delay * 1000;
     pidRuntime.crashRecoveryAngleDeciDegrees = pidProfile->crash_recovery_angle * 10;
     pidRuntime.crashRecoveryRate = pidProfile->crash_recovery_rate;
-    pidRuntime.crashGyroThreshold = pidProfile->crash_gthreshold;
-    pidRuntime.crashDtermThreshold = pidProfile->crash_dthreshold;
+    pidRuntime.crashGyroThreshold = pidProfile->crash_gthreshold; // error in deg/s
+    pidRuntime.crashDtermThreshold = pidProfile->crash_dthreshold * 1000.0f; // gyro delta in deg/s/s divided by 1000 to match origingal 2017 intent
```

Comment:
```suggestion
    pidRuntime.crashDtermThreshold = pidProfile->crash_dthreshold * 1000.0f; // gyro delta in deg/s divided by 1000 to match original 2017 intent
```

Reply:
@ctzsnooze - please address so this can be merged.

Have tested on PAVO25 and works perfectly (using low value of 20).
Could not test on BETAFPVF4SX1280 as after enabling airmode there is no RC input using ELRS SPI (same on master)

Reply:
@haslinghuis - fixed, and comment fixed also.  Thank you for picking this up.
Would be great to get this merged!